README.md

```diff
@@ -13,12 +13,15 @@ You will need the following things properly installed on your computer.
 
 * [Git](http://git-scm.com/)
 * [Node.js](http://nodejs.org/) (with NPM)
+* [Yarn](https://yarnpkg.com/)
 * [Bower](http://bower.io/)
 * [Ember CLI](http://ember-cli.com/)
 * [PhantomJS](http://phantomjs.org/)
 
 ## Installation
-* `npm install`
+
+* `git clone` this repository
+* `yarn`
```

Comment:
Yarn wants you to use `yarn`, the default command for install (it's more of their convention than a convenience), but I won't bikeshed this. Changed it in the latest commit.

I changed everything to use --pure-lockfile as you suggested.